src/LinkProcessor/PresenterFactoryFaker.php

```diff
@@ -4,25 +4,42 @@
 
 namespace Efabrica\PHPStanLatte\LinkProcessor;
 
+use InvalidArgumentException;
+use Nette\Application\IPresenterFactory;
 use Nette\Application\PresenterFactory;
 
 final class PresenterFactoryFaker
 {
     /** @var array<string, string> */
     private array $mapping;
 
+    private ?string $presenterFactoryBootstrap;
+
+    private ?IPresenterFactory $presenterFactory = null;
+
     /**
      * @param array<string, string> $mapping
      */
-    public function __construct(array $mapping)
+    public function __construct(array $mapping, ?string $presenterFactoryBootstrap)
     {
         $this->mapping = $mapping;
+        $this->presenterFactoryBootstrap = $presenterFactoryBootstrap;
     }
 
-    public function getPresenterFactory(): PresenterFactory
+    public function getPresenterFactory(): ?IPresenterFactory
     {
-        $presenterFactory = new PresenterFactory();
-        $presenterFactory->setMapping($this->mapping);
-        return $presenterFactory;
+        $presenterFactory = null;
```

Comment:
If presenter factory is already created it should be returned amd not created again

Reply:
Yeah, I have presenterFactory property, but only initialized, not used 🤦